### PR TITLE
New experiment 2: Pleiades Work-Precision diagram

### PR DESCRIPTION
**Pull request by pnkraemer**
This PR introduces the code that leads to a successfull pleiades WP diagram. 

It relies on the most recent jitting branches in tornadox.

### CODE REVIEW

File: experiments/2_medium_scale_problem.py [hunk 1]
```diff
@@ -12,19 +12,21 @@
...
+# IVP = tornadox.ivp.pleiades()
+IVP = tornadox.ivp.fhn_2d(bbox=[[-1.0, -1.0], [1.0, 1.0]], dx=0.5)
 IVP = tornadox.ivp.pleiades()
-# IVP = tornadox.ivp.lorenz96()
 reference_sol = solve_ivp(
     fun=IVP.f,
     t_span=(IVP.t0, IVP.tmax),
     y0=IVP.y0,
     method="LSODA",
-    jac=IVP.df,
```
Comment (nathanaelbosch) [inline experiments/2_medium_scale_problem.py:22]: In my experience this often made LSODA much faster!

Comment (nathanaelbosch) [reply experiments/2_medium_scale_problem.py:22]: But I saw that you have this enabled in the actual scipy experiments, so this should not be an issue here :)

Comment (pnkraemer) [reply experiments/2_medium_scale_problem.py:22]: To be honest, I did not know that LSODA uses Jacobians... and I also dont remember deleting this line, sorry! That was not very bright of me. I added it back in. 

(Even though this is just the reference solution...:))

File: src/hose/plotting.py [hunk 2]
```diff
@@ -46,33 +47,38 @@
...
 MATCH_STYLE = {
-    "KroneckerEK0": (EK0_color, "dotted", "o"),
-    "ReferenceEK0": (EK0_color, REF_LINESTYLE, "^"),
-    "DiagonalEK0": (EK0_color, "-.", "s"),
-    "ReferenceEK1": (EK1_color, REF_LINESTYLE, "d"),
-    "DiagonalEK1": (EK1_color, "-.", "s"),
-    "TruncationEK1": (EK1_color, "dashed", "p"),
-    "EarlyTruncationEK1": (EK1_color, "dotted", "^"),
-    "EnK1": (EnK_color, "dashed", "s"),
+    "KroneckerEK0": (EK0_color, "dotted", "o", 0.9, THICK),
```
Comment (nathanaelbosch) [inline src/hose/plotting.py:70]: Do the previous plots still run fine with this change? Just to make sure that none of the previous scripts gets destroyed

File: experiments/2_medium_scale_problem.py [hunk 3]
```diff
@@ -106,36 +189,43 @@ def time_function(self, fun):
...
-        rtol=rtol,
-        num_derivatives=nu,
-    )
-    for (alg, (atol, rtol), nu) in itertools.product(
-        ALGS, zip(ATOLS, RTOLS), NUM_DERIVS
-    )
-]
+ALGS_SCIPY = ["RK45", "Radau"]
+
+ATOLS = 1 / 10 ** jnp.arange(3, 13)
```
Comment (nathanaelbosch) [inline experiments/2_medium_scale_problem.py:201]: Does it make sense to go until 13, when the reference is at 12?

**Review by nathanaelbosch — COMMENTED** [review]
2 comments,

**Review by nathanaelbosch — APPROVED** [review]

File: src/hose/plotting.py (hunk 2 above)
Comment (pnkraemer) [reply src/hose/plotting.py:70]: (as discussed offline, they do)

File: experiments/2_medium_scale_problem.py (hunk 3 above)
Comment (pnkraemer) [reply experiments/2_medium_scale_problem.py:201]: I think so; the RMSE will taper off (which is not bad!), and who says that all the solvers reach all the tolerances reliably :)

**Pull request merged**